1_antenatal_care.do

```diff
@@ -61,7 +65,7 @@ order *,sequential
 	replace c_anc_ski_q = . if mi(c_anc_ski) & c_anc_any == 1
 	
 	*c_anc_bp: Blood pressure measured during pregnancy of births in last 2 years
-	clonevar c_anc_bp = m42c 
+	clonevar c_anc_bp = m42c            // please check this as well as bs and ur with my codes

```

Comment:
Are you saying this might be country specific?

Reply:
I found a filter question in recode seven. It may be the same for recode six. Please check. And if so, you may need to update the code.

Reply:
Do you change anything on the codes?